server.js

```diff
@@ -5,16 +5,16 @@ var fs         = require('fs');
 var SNSClient  = require('aws-snsclient');
 var winston    = require('winston');
 
-winston.add(winston.transports.File, { filename: '/app/log/ses_json.log' });
-winston.remove(winston.transports.Console);
+const logFile = new winston.transports.File({ filename: './log/ses_json.log' });
+winston.add(logFile);
```

Comment:
The [default Winston logger](https://github.com/winstonjs/winston#using-the-default-logger) has no transports configured, so we shouldn't need to remove the console any more.

Reply:
For some reason, when I tried to run on docker, I got an error on docker-compose up 

```
email-verify_1  | /app/node_modules/winston/lib/winston/logger.js:481
email-verify_1  |   var instance = created ? transport : (new (transport)(options));
email-verify_1  |                                         ^
email-verify_1  | 
email-verify_1  | TypeError: transport is not a constructor
```

https://github.com/winstonjs/winston/issues/1094

This ended up working. 
```suggestion
winston.add( winston.transports.File, { filename: './log/ses_json.log' });
```

Reply:
That’s the syntax for Winston v2 but this PR upgrades to v3.

Reply:
See the changes to `winston.Logger.add` here:
https://github.com/winstonjs/winston/blob/master/UPGRADE-3.0.md#winstonlogger

Reply:
ooh. i probably built from master then went to the PR to test. my bad.